packages/forms/src/rhf/fields/Select/RHFSelect.component.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useFormContext } from 'react-hook-form';
+import get from 'lodash/get';
+
+import Select from '../../../widgets/fields/Select';
+
+function RHFSelect(props) {
```

Comment:
You mut add your RHFSelect component in src/rhf/index to be able to import it in UMD mode. 
The import will be `import Form from '@talend/react-forms'` then yo ucan use it via `Form.rhf.Select`